scala-native-cli-integration-test-runner/src/test/run/integration-test/test

```diff
@@ -0,0 +1,31 @@
+> compile
+
+# Print used version
+> runCli --version
+
+# -- Link and check if outpath exists
+> runCli --outpath target/out1 -v -v Main$ 
+$ exists target/out1
+
+# -- Fail to link without main specified
+> runCli --outpath target/out2 -v -v
+-$ exists target/out2
+
+# -- Fail to link with an incorrect option
+> runCli --outpath target/out3 -v -v --gc fast Main$ 
```

Comment:
I've played with it a bit, and I think we should have a mandatory `--main` option, while positional arguments should be used only for classpath.  
Btw. I've found a bug in Scala Native, where you cannot use a single file in a linker - instead, you need to pass the whole directory or jar

Reply:
I imagine VirtualDirectory.real() may be the culprit here, I think that is how classpath is being parsed